### ISSUE TITLE
Unknown field in stream descriptor ("dash_only")?!

### ISSUE THREAD

**Issue by AradAral** [google/shaka-packager#950]
## System info
**Operating System**: Windows 10, build 19042.985
**Shaka Packager Version**: 2.4.3 (currently latest)

## Issue and steps to reproduce the problem

**Packager Command**:

```bash
packager ^
'in=input/720p.mp4,stream=video,init_segment=output/720p/init.mp4,segment_template=output/720p/seg-$Number$.m4s,playlist_name=output/720p/playlist.m3u8' ^
'in=input/480p.mp4,stream=video,init_segment=output/480p/init.mp4,segment_template=output/480p/seg-$Number$.m4s,playlist_name=output/480p/playlist.m3u8'  ^
'in=input/360p.mp4,stream=video,init_segment=output/360p/init.mp4,segment_template=output/360p/seg-$Number$.m4s,playlist_name=output/subtitle/playlist.m3u8'  ^
'in=input/audio.mp4,stream=audio,init_segment=output/audio/init.mp4,segment_template=output/audio/seg-$Number$.m4s,playlist_name=output/audio/playlist.m3u8'  ^
'in=input/subtitle.vtt,stream=text,init_segment=output/subtitle/init.mp4,segment_template=output/subtitle/seg-$Number$.m4s,dash_only=1'  ^
'in=input/subtitle.vtt,stream=text,segment_template=output/subtitle/seg-$Number$.vtt,playlist_name=output/subtitle/playlist.m3u8,hls_only=1' ^
--generate_static_live_mpd ^
--hls_master_playlist_output output/master.m3u8 ^
--mpd_output output/manifest.mpd
```

### What is the expected result?
It should generate `.vtt` subtitle segments for HLS, and `.m4s` for DASH, note that I'm doing pretty much the exact same thing as [this example](https://google.github.io/shaka-packager/html/tutorials/dash.html) in the tutorial, but strangely enough, I get an error saying that there's no option called `dash_only`! The same thing happens with `hls_only`.

### What happens instead?
The following error is thrown:
```
[0531/134909:ERROR:stream_descriptor.cc(210)] Unknown field in stream descriptor ("dash_only").
```

I've tried a thousand different DASH/HLS packagers, and Shaka Packager has been the absolute best among them all. I also very much needed this exact feature, to be able to create `.vtt` segments for HLS and `.m4s` for DASH, so I was absolutely thrilled to read the tutorial and discover that Shaka supports this.

What am I doing wrong? Thanks.

**Comment by AradAral** [comment]
**UPDATE:** By exploring the source code of version 2.4.3 (latest) I found out that the code related to the `dash_only` and `hls_only` options aren't there. But then I saw that the relevant pull request (#721) was merged on March, 6th, 2020.

So, it's been basically more than 1 year and 2 months since that pull request was merged but we still don't have a release that includes it.

Approximately when can we expect such a release?
